overseer.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 const (
-	envSlaveID        = "OVERSEER_SLAVE_ID"
-	envIsSlave        = "OVERSEER_IS_SLAVE"
+	envChildID        = "OVERSEER_CHILD_ID"
+	envIsChild        = "OVERSEER_IS_CHILD"
```

Comment:
i think this change is the only one that will cause issues - this variable is used between both processes, so if a user updates their app with the latest overseer, the newly created child process will have the old variables set (by the old parent process), breaking the app

i think the least risky thing to do is just keep these 2 strings... the alternative is to come up with a transition version, which supports both old/new variable names

Reply:
Makes sense... But if you make that argument you could never change it as there could always be someone with an old version.
I assume you don't want to do a v2 release just for this?
I'd change it with a transition approach if you decide that to be the right way.